test/translate-test.js

```diff
@@ -10,16 +10,62 @@ describe('translateResponse', () => {
   beforeEach(() => {
     patientData = JSON.parse(patientDataContent).patientData;
     customApiResponse = JSON.parse(resultContent);
-    translateResponse(customApiResponse, patientData);
   });
 
   describe('parse procedure data', () => {
     it('should include Cervix Excision procedure', function () {
+      translateResponse(customApiResponse, patientData);
       expect(patientData.length).to.be.greaterThan(2);
 
       const procedures = patientData.filter(pd => pd.resourceType === 'Procedure');
-      expect(procedures.length).to.be.greaterThan(0);
+      expect(procedures.length).to.be.greaterThan(1);
       expect(procedures.some(p => p.code.coding.some(coding => coding.code === '120038005'))).to.be.true;
     });
   });
+
+  describe('Parse excision results that are not AIS or histologic cancer', () => {
+    it('should ignore excision results that do not show AIS or histologic cancer', function () {
+      translateResponse(customApiResponse, patientData);
+      const diagnosticReports = patientData.filter(pd => pd.resourceType === 'DiagnosticReport');
+
+      expect(diagnosticReports.length).to.equal(1);
+      expect(diagnosticReports.some(p => p.code.coding.some(coding => coding.code === '65753-6'))).to.be.false;
+    });
+  });
+
+  describe('Parse AIS excision results', () => {
```

Comment:
These three describes can be merged into one which has three sub 'it' test, like
```
describe('Parse excision results', () => {
  it('should ignore excision results that do not show AIS or histologic cancer', () => {
    ...
  });
  it('should map an AIS excision result to a biopsy', () => {
    ...
  });
  it('should map a histologic cancer excision result to a biopsy', () => {
  });
});

Reply:
Good idea - I like that better. I'll push a commit shortly to clean this up.

Reply:
Fixed: https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/20/commits/72b8fd505f8bd089948b6aa3e8cd498ad0bfc1b0